doc/man3/X509_set0_libctx.pod

```diff
@@ -0,0 +1,56 @@
+=pod
+
+=head1 NAME
+
+X509_set0_libctx - set X509 certificates library context
+
+=head1 SYNOPSIS
+
+ #include <openssl/x509.h>
+
+ void X509_set0_libctx(X509 *x509, OPENSSL_CTX *libctx, const char *propq);
+
+=head1 DESCRIPTION
+
+X509_set0_libctx() may be called before calling other X509 functions that need 
+a library context I<libctx> and property query I<propq> to fetch algorithms
+from providers. These values are both NULL by default which results in the
+default (NULL) library context being used.
+
+=head1 NOTES
+
+The X509_set0_libctx() should be used by certificate objects if the library
+context is required. The following functions use the values set by
```

Comment:
Compare with X509v3_cache_extensions. I don't think we need both of those functions.

Reply:
Removed X509v3_cache_extensions()